KS-4-5 задача 3

VVShchegolev opened the pull request.

Commit:
KS-4-5 задача 3

ievetrovdev commented on src/main/kotlin/Lesson_4_Ex_3/Lesson_4_Ex_3.kt:
```diff
@@ -0,0 +1,14 @@
+    val humidity: Int = 20
+    val season: String = "зима"
+
+    val areConditionsFavorable = isSunny && isTentOpen && humidity == 20 && season != "зима"
```
Значения в этом коде являются [магическими числами](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)#%D0%9F%D0%BB%D0%BE%D1%85%D0%B0%D1%8F_%D0%BF%D1%80%D0%B0%D0%BA%D1%82%D0%B8%D0%BA%D0%B0_%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F). Стоит вынести в константы критерии, с которыми сравниваем.

ievetrovdev commented:
Это задача `KS-4-3`, а не `KS-4-5`

VVShchegolev committed:
Update KS_4_3 Задача 3 исправлена

Что было исправлено:
"Магическое число" и "Магическая строка" были вынесены в константы

ievetrovdev reviewed (approved):
Отлично!
